sdk/tests/client/signing/account.rs

```diff
@@ -110,9 +110,9 @@ async fn sign_account_state_transition() -> Result<()> {
         &tx_payload,
         prepared_transaction_data.mana_rewards,
         protocol_parameters,
-    )?;
+    );
```

Comment:
Just do this and remove the if let Err() below?
```suggestion
    )?;
```

Reply:
Same in many other tests or with unwrap

Reply:
I guess it was done this way to include the `tx_payload` but I don't think it's really necessary in tests so yeah I can remove

Reply:
Done

Reply:
Yeah usually shouldn't be needed there and if it panics we can add it just to investigate